src/data/vapor-pressure-constants.ts

```diff
@@ -2748,6 +2748,891 @@ export const DIPPRVaporPressureDictionary: Record<
     pressureAtMinimumTemperature: 7.21e3,
     pressureAtMaximumTemperature: 1.315e6,
   },
+
+  HydrogenBromide: {
+    name: 'Hydrogen bromide',
+    formula: 'BrH',
+    CAS: '10035-10-6',
+    C1: 29.315,
+    C2: -2424.5,
+    C3: -1.1354,
+    C4: 2.38e-18,
+    C5: 6,
+    minimumTemperature: 185.15,
+    pressureAtMinimumTemperature: 29500,
+    maximumTemperature: 363.15,
+    pressureAtMaximumTemperature: 8460000,
```

Comment:
Sir @maneetgoyal  during automation from table 2 fields are not consisitent like maximum temp below minimum one is it acceptabale or should i fix it